src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -171,6 +171,14 @@ public String getS3Path() {
         return getString("secor.s3.path");
     }
 
+    public String getS3MigrationPath() {
```

Comment:
New properties needs to have a line in src/main/config/secor.common.properties.

Maybe the property name can be called S3AlternatePath and S3AlthertePathDate